### PR TITLE
[SW-872] Refactor: Migrate tokens component to TypeScript and Composition API

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/Tokens.vue [hunk 1]
```diff
@@ -76,25 +85,29 @@ export default defineComponent({
...
-      if (props.tokens?.length < 2) return props.symbolLength;
-      const shorterNameLength = [props.tokens[0].symbol.length, props.tokens[1].symbol.length]
-        .find((length) => length < props.doubleSymbolLength);
+      if (props.tokens?.length < 2) {
+        return props.symbolLength;
+      }
+      const shorterNameLength = [
+        props.tokens[0].symbol.length,
+        props.tokens[1].symbol.length,
+      ].find((length) => length < props.doubleSymbolLength);
```
Comment (pstachel) [inline src/popup/components/Tokens.vue:94]: What are your thoughts of removing that additional assignment?
```suggestion
      const shorterNameLength = props.tokens.find(({ symbol }) => symbol.length < props.doubleSymbolLength);
```

**Review by pstachel — APPROVED** [review]

File: src/popup/components/Tokens.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/Tokens.vue]: OK, makes sense. But I did it in a little different way - mapping the tokens and then finding.

File: src/popup/pages/FungibleTokens/TokenContainer.vue [hunk 2]
```diff
@@ -166,33 +168,35 @@ export default defineComponent({
     const loading = ref<boolean>(true);
     const tokenPairs = ref({ token0: null, token1: null });
     const tokenBalances = useGetter<any[]>('fungibleTokens/tokenBalances');
-    const availableTokens = computed(() => root.$store.state.fungibleTokens.availableTokens);
+    const availableTokens = computed(
+      (): ITokenList => root.$store.state.fungibleTokens.availableTokens,
+    );
```
Comment (pstachel) [inline src/popup/pages/FungibleTokens/TokenContainer.vue:173]: If you'd like to you could use `useState` hook

**Review by pstachel — APPROVED** [review]

File: src/popup/pages/FungibleTokens/TokenContainer.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/pages/FungibleTokens/TokenContainer.vue]: Done

**Pull request merged**